build.cake

```diff
@@ -94,10 +94,10 @@ Task("Package")
     .IsDependentOn("Test")
     .Does(() => {
         CreateDirectory(buildDir);
-        CreateDirectory(Path.Combine(buildDir, "content"));
-        CreateDirectory(Path.Combine(buildDir, "content", "net35"));
-        CreateDirectory(Path.Combine(buildDir, "content", "net40"));
-        CreateDirectory(Path.Combine(buildDir, "content", "netcore45"));
+        CreateDirectory(Path.Combine(buildDir, "lib"));
```

Comment:
Looking at the [multi-targeting documentation](https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks#framework-version-folder-structure), I'm wondering if the `lib` folder has the same effect as the old `content` folder? Wonder if we need these at all.

Reply:
In this case, yes, the `lib` folder replaces what the `content` folder provided at the time. I'm not sure if you can just do away with the `lib` folder or not. But this is saying that this package can be installed into projects targeting .NET 3.5, 4.0, and Windows 8 "store" apps.